src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+
+        Colors color = ColorSupplier.getRandomColor();
+        int number = new Random().nextInt(100) + 1;
```

Comment:
100 is magic number
![image](https://user-images.githubusercontent.com/12864128/220933253-642b95f4-c94b-45df-871b-084a226eb567.png)